test/unit/controllers/components/create-model-form.js

```diff
@@ -43,9 +43,9 @@ describe('ModelFormController', () => {
                 })
             );
 
-            ModelFormController.createModel('OIUIO').then(
-                assert.fail
-            ).catch((e) => {
+            return ModelFormController.createModel('OIUIO').then(() => {
+                assert.fail;
```

Comment:
When this entire `.then()` is removed, the test still passes, so there is code in our test that does absolutely nothing. Looking at the source, I'm thinking we don't even need the `.then()` anyway.